frontend/src/pages/ActivityReport/index.js

```diff
@@ -86,9 +89,9 @@ function ActivityReport({ match, user, location }) {
         updateLoading(true);
 
         const apiCalls = [
-          getRecipients(),
-          getCollaborators(region),
-          getApprovers(region),
+          getRecipients(regionIdToUse),
+          getCollaborators(regionIdToUse),
+          getApprovers(regionIdToUse),
```

Comment:
There are two cases we need to handle for this component

1. The report is new
2. The report has already been created

In the first case we want to use `regionIdToUse`. In the second we want to use the region of the report. We will want to refactor this `useEffect` a little. The basic gist would be to do the `if (activityReportId !== 'new) {` check first in this function and use `regionIdToUse` if the report is new or `report.region` if the report is not new.